modules/vba_documents/app/views/vba_documents/monthly_report_mailer/_monthly_processing_success.html.erb

```diff
@@ -1,29 +0,0 @@
-<%
```

Comment:
This partial wasn't called from anywhere in the app, and I learned after a quick investigation that it was for a feature that was never completed, so I removed it.